src/schemas/profile.json

```diff
@@ -51,6 +51,12 @@
           "title": "farcaster",
           "pattern": "^[a-z0-9-]*$",
           "maxLength": 17
+        },
+        "emailSubscriptions": {
+          "type": "array",
+          "title": "email subscriptions",
+          "uniqueItems": true,
+          "items": { "type": "string" }
```

Comment:
This can be sent with `subscribtion` no? do we need it on profile?

Reply:
This is for the edition, so that any changes are saved on IPFS

Anything send with `Subscription` will not be saved on ipfs, since containing the email address

Reply:
Users won't be able to update their email? we can add these to `Subscription` and make email optional right?

Reply:
Updating email => send email with Subscription again

If we add `emailSubscriptions` array to `Subscription`, (which make more sense), we would also skip saving those update on ipfs (not a big deal I guess, means that anything related to subscription will not be saved and tracked on ipfs).

Reply:
Moved to `Subscription`, into the `metadata` field